File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/IPinCheerleader.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class IPinCheerleader : StoryChoice
	{
		internal IPinCheerleader() : base(StoryChoice.GetLocalizedString("ch5IPinCheerleader"), 5070)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5SparredWithCheerleader = CheerleaderSpar.Pinned;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/IPinCheerleader.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.CheerleaderSpar;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class IPinCheerleader extends StoryChoice {
    public IPinCheerleader() {
        super(R.string.ch5IPinCheerleader, 5070);
    }

    @Override
    public void select()
    {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5SparredWithCheerleader(CheerleaderSpar.Pinned);
        super.select();
    }
}
